dbt/include/databricks/macros/materializations/incremental/incremental.sql

```diff
@@ -3,6 +3,7 @@
   {#-- Validate early so we don't run SQL if the file_format + strategy combo is invalid --#}
   {%- set raw_file_format = config.get('file_format', default='delta') -%}
   {%- set raw_strategy = config.get('incremental_strategy') or 'merge' -%}
+  {%- set incremental_predicates = config.get('incremental_predicates', default=none) -%}
```

Comment:
nit: Shall we move this line to the next line of `{%- set incremental_strategy = ... -%}`?

Reply:
@ueshin just to confirm, you'd prefer the predicates line on line 10?

Reply:
Yes, I'd prefer to put it in the same block as the other configs, not the temporary ones.